app/Repositories/ReportbackRepository.php

```diff
@@ -179,11 +181,48 @@ public function updateReportbackItems($data)
         foreach ($data as $reportbackItem) {
             if (isset($reportbackItem['rogue_reportback_item_id']) && ! empty($reportbackItem['rogue_reportback_item_id'])) {
                 $rbItem = ReportbackItem::where(['id' => $reportbackItem['rogue_reportback_item_id']])->first();
+                $rb = Reportback::where(['id' => $rbItem->reportback_id])->first();
 
                 if ($reportbackItem['status'] && ! empty($reportbackItem['status'])) {
+                    // @TODO: update to add more details in the event e.g. admin who reviewed, admin's northstar id, etc.
+                    $reportbackItem['submission_type'] = 'admin';
+
+                    // Create the Event.
+                    $event = Event::create([
+                        'signup_id' => 0,
+                        // Do we want the user's northstar id or the admin's?
+                        'northstar_id' => $rb->northstar_id,
```

Comment:
@sbsmith86 @katiecrane @DFurnes same with this! Should we change it to the admin's northstar id?